controllers/photoController.ts

```
@@ -0,0 +1,16 @@
+import { getRoverPhotos } from "../clients/nasaClient"
+import {Photo} from "../models/roverData"
+
+export const PhotoController = {
+    Index: async (req: any, res: any) => {
+        try {
+            const photoData: Photo[] | void = await getRoverPhotos(req.params, req.query)
```

Biome

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)